var/spack/repos/builtin/packages/apcomp/package.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+import socket
+import llnl.util.tty as tty
+
+
+def cmake_cache_entry(name, value, vtype=None):
+    """
+    Helper that creates CMake cache entry strings used in
+    'host-config' files.
+    """
+    if vtype is None:
+        if value == "ON" or value == "OFF":
+            vtype = "BOOL"
+        else:
+            vtype = "PATH"
+    return 'set({0} "{1}" CACHE {2} "")\n\n'.format(name, value, vtype)
+
+
+class Apcomp(Package):
+    """A multi use-case image compositor"""
+
+    homepage = 'https://github.com/Alpine-DAV/ap_compositor'
+    git      = 'https://github.com/Alpine-DAV/ap_compositor.git'
+    url      = "https://github.com/Alpine-DAV/ap_compositor/releases/download/v0.0.1/apcomp-v0.0.1.tar.gz"
+
+    maintainers = ['mclarsen','cyrush']
```

Comment:
PEP8 probably needs this:
```suggestion
    maintainers = ['mclarsen', 'cyrush']
```